Marlin/src/lcd/ultralcd.cpp

```diff
@@ -976,13 +976,16 @@ void lcd_quick_feedback(const bool clear_buttons) {
 
   #endif // POWER_LOSS_RECOVERY
 
-  #if ENABLED(SINGLENOZZLE)
```

Comment:
Making `M217` always enabled even on machines that have only a single extruder makes no sense.